config.json

```diff
@@ -961,6 +961,18 @@
         "loops"
       ]
     },
+    {
+      "slug": "zebra-puzzle",
+      "uuid": "b1e2bd39-3f4b-44c1-b7e2-258d4ee241f8",
+      "core": false,
+      "unlocked_by": "saddle-points",
+      "difficulty": 6,
+      "topics": [
+        "arrays",
+        "conditionals",
```

Comment:
I'm not sure I would add `arrays` and `conditionals`. You don't necessarily need to use `arrays` and while you probably would use `conditionals` I'm not sure it's the main focus of the exercise. So I think it's fine just to have `logic` as the topic as it is very much an exercise about logic 🙂 What do you think @sjwarner-bp @lemoncurry? 🙂